android/expoview/src/main/java/host/exp/exponent/LauncherActivity.java

```diff
@@ -82,6 +84,7 @@ public void onNewIntent(Intent intent) {
 
     // We shouldn't ever get here, since we call finish() in onCreate. Just want to be safe
     // since this Activity is singleTask and there might be some edge case where this is called.
+    UserInteractionReceiver.getInstance().onIntent(intent, this);
```

Comment:
I understand you needed to put this here to know in `UserInteractionReceiver` which Activity to start when user interacts with a notification. Instead of putting this code in specific activities maybe we could add this configuration to `AndroidManifest`? Somehow specify that 'this' activity should be started when user interacts with a notification?